code/modules/fishing/fish/_fish.dm

```diff
@@ -180,29 +206,30 @@
 	if(!istype(item, /obj/item/fish_feed))
 		return ..()
 	if(!item.reagents.total_volume)
-		balloon_alert(user, "[item] is empty!")
+		balloon_alert(user, "[item.name] is empty!")
 		return TRUE
 	if(status == FISH_DEAD)
-		balloon_alert(user, "[src] is dead!")
+		balloon_alert(user, "[name] [HAS_MIND_TRAIT(user, TRAIT_NAIVE) ? "isn't hungry" : "is dead!"]")
 		return TRUE
 	feed(item.reagents)
-	balloon_alert(user, "fed [src]")
+	balloon_alert(user, "fed [name]")
```

Comment:
why the src -> name

Reply:
"fed slimefish" vs "fed the slimefish", it's more compact this way, since balloon alert last roughly a second before fading out, therefore there's not much room for text.

Reply:
wouldn't \improper solve this? (not rhetorical I'm not actually sure if it would) https://www.byond.com/docs/ref/#/DM/text/macros

Reply:
Improper just treats capitalized nouns as uncapitalized, and ironically it prob does the opposite: "you hug the John Doe" vs "you hug John Doe".